src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -221,6 +226,16 @@ private User getUser(String username) {
                 getMessageSource(ResponseMessagesHandler.ExceptionMessage.USER_NOT_FOUND)));
     }
 
+    private void checkPasswordUniqueAndNotUsed(String userPassword, String newPassword, String confirmPassword)
+            throws DataConflictException {
+        if (userPassword.equals(newPassword)) {
```

Comment:
For a single check we don't need to create a new method

Reply:
++